### ISSUE TITLE
shaders on optifine/optifabric conflict with origins-plus

### ISSUE THREAD

**Issue by simplyskellix** [lochnessdragon/OriginsPlus#3]
when using this mod you are unable to use shaders in optifine/optifabric. optifabric says that I have satin API downloaded whenever I add origins-plus and wont allow shaders, I have no clue how or why this would happen and since the game does not crash I have no crash report to show how this could be happening. I myself do not code or understand it what so ever so please do excuse how little information I can give.

mod conflict names and versions
+origins plus 0.0.2
+optifine G6
+optifabric 1.9.6
+any related dependencies for functionality

**Comment by lochnessdragon** [comment]
Yes. Thank you. We use Satin in the backend for the dwarven. I will look into fixing that.

**Comment by Foxicer** [comment]
i would love it if shaders would actually work

**Comment by OneTripleSSS** [comment]
Uhm I apologize for bugging you about it, but... I am looking to add this to my server but people want to use shaders as well. and my origin has code that is based off of your mod. thanks in advance! 
-Trip
